### ISSUE TITLE
Group 2 

### ISSUE THREAD

**Issue by LauraKoehn-NOAA** [nmfs-openscapes/2022-noaa-nwfsc-fall#11]
Diana and Chris, what's your favorite animal?

- [x] Task - favorite animal? @DianaDishman-NOAA 
- [ ] Task - favorite animal? @chris3jordan

**Comment by ericward-noaa** [comment]
What kind of issue is this? Bug, etc?

**Comment by ericward-noaa** [comment]
Also -- try assigning it to someone (I nominate Chris!)

**Comment by DianaDishman-NOAA** [comment]
Mantis shrimp
[https://theoatmeal.com/comics/mantis_shrimp](url)

**Comment by EmilyMarkowitz-NOAA** [comment]
![giphy](https://user-images.githubusercontent.com/75965120/197273635-7404d9d5-5f29-4ac7-b89b-31d022a470a9.gif)

**Comment by LauraKoehn-NOAA** [comment]
Yes!! Both great! just learned about the piglet squid and I love it.

**Comment by DianaDishman-NOAA** [comment]
Had never heard of this - amazing!
